torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -158,7 +158,11 @@ def tuned_mm_plus_mm(mat1, mat2, mat3, mat4, *, layout=None):
 
     assert layout1 == layout2
     # options to tune from
-    choices = [aten_mm_plus_mm.bind((mat1, mat2, mat3, mat4), layout1)]
+    choices = (
+        [aten_mm_plus_mm.bind((mat1, mat2, mat3, mat4), layout1)]
+        if use_aten_gemm_kernels
+        else []
+    )
     if use_triton_template(layout1):
```

Comment:
If `use_triton_template()` is false (which happens for CPU kernels), then choices will be empty and we will hit an error.

Same in other files.